config/jobs/kubernetes/test-infra/test-infra-trusted.yaml

```diff
@@ -71,6 +71,38 @@ postsubmits:
       - name: creds
         secret:
           secretName: pusher-service-account
+  kubernetes/k8s.io:
+  - name: cip-prodrun
+    cluster: test-infra-trusted
+    decorate: true
+    run_if_changed: ".*/manifest.yaml"
+    skip_report: false
```

Comment:
Don't we want to leave a note on the commit saying whether the job passed?

Reply:
I thought that's what `skip_report: false` did already?